extensions/analyticsdx-vscode-core/package.nls.json

```diff
@@ -7,6 +7,8 @@
   "analyticsdx_app_create_label": "SFDX: Create Analytics App From Template",
   "analyticsdx_app_delete_label": "SFDX: Delete Analytics App",
 
+  "analyticsdx_dashboard_lwc_create_label": "SFDX: Create Analytics Dashboard LWC",
```

Comment:
@mblumreich, @skipsauls FYI - This is what will show in the command palette and the right-click menu.